.gitignore

```diff
@@ -2,4 +2,5 @@
 /bin/php-cs-fixer
 /bin/phpspec
 /vendor
+.php_cs.cache
```

Comment:
I would use `.*.cache` instead, to catch phpunit cache too

Reply:
In this library there aren't phpunit! :P